doc/installation-and-configuration.md

```diff
@@ -74,7 +74,7 @@ While preparing a Virtual Machine that will host SonarQube database, portal and/
 	- **>> NOTE >>** This walkthrough assumes the use of the BK VM. If, for example, you are using **SQLExpress** instead, you have to update the connection string. Example:
 
 		```
-		sonar.jdbc.url=jdbc:jtds:sqlserver://localhost/Sonar;instance=SQLEXPRESS;SelectMethod=Cursor
+		sonar.jdbc.url=jdbc:sqlserver://localhost;databaseName=sonar;integratedSecurity=true

```

Comment:
How does the SQLExpress version of the connection string differ from the non-SQLExpress version?
The updated version of the connection string no longer specifies the SQL instance. Is this correct?